homeassistant/components/tolo/select.py

```diff
@@ -27,7 +27,6 @@ async def async_setup_entry(
 class ToloLampModeSelect(ToloSaunaCoordinatorEntity, SelectEntity):
     """TOLO Sauna lamp mode select."""
 
-    _attr_device_class = "tolo__lamp_mode"
```

Comment:
It does exist: <https://github.com/home-assistant/core/blob/dev/homeassistant/components/tolo/strings.select.json#L3>